pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -628,10 +628,13 @@ def _render_jira_template(fields: Dict, template_vars: Dict):
         return fields
 
     async def set_advisory_dependencies(self, advisories):
-        blocking_kind = ['image', 'extras']
-        target_kind = ['rpm', 'metadata']
-        expected_blocking = {i for i in [advisories[k] for k in blocking_kind if k in advisories] if i}
-        target_advisories = {i for i in [advisories[k] for k in target_kind if k in advisories] if i}
+        dependencies = {
+            'image': {'rpm', 'metadata'},
+            'extras': {'rpm', 'metadata'},
+            'rpm': {'microshift'},
+        }
+        expected_blocking = {i for i in [advisories[k] for k in dependencies.keys() if k in advisories] if i}
+        target_advisories = {i for i in [advisories[k] for k in set.union(*dependencies.values()) if k in advisories] if i}
```

Comment:
This will cause `rpm` `metadata` and `microshift` advisories all ship after `{'rpm', 'image', 'extras'}`, which doesn't reflect the intent and make `rpm` depend on itself.